resources/js/views/settings/serverPools/Index.vue

```diff
@@ -0,0 +1,287 @@
+            <b-button
+              v-b-tooltip.hover
+              :title="$t('settings.serverPools.edit', { name: data.item.name })"
+              :disabled='isBusy'
+              variant='dark'
+              :to="{ name: 'settings.server_pools.view', params: { id: data.item.id } }"
+            >
+              <i class='fas fa-edit'></i>
+            </b-button>
+          </can>
+          <can method='delete' :policy='data.item'>
+            <b-button
+              v-b-tooltip.hover
+              :title="$t('settings.serverPools.delete.item', { name: data.item.name })"
+              :disabled='isBusy'
+              variant='danger'
+              @click='showServerPoolModal(data.item)'>
+              <i class='fas fa-trash'></i>
+            </b-button>
+          </can>
+        </b-button-group>
+      </template>
+    </b-table>
+
+    <b-pagination
+      v-model='currentPage'
+      :total-rows='total'
+      :per-page='perPage'
+      aria-controls='server-pools-table'
+      @input="$root.$emit('bv::refresh::table', 'server-pools-table')"
+      align='center'
+      :disabled='isBusy'
+    ></b-pagination>
+
+    <b-modal
+      :busy='deleting'
+      :hide-footer="deleteFailedRoomTypes!=null"
+      ok-variant='danger'
+      cancel-variant='dark'
+      :cancel-title="$t('app.no')"
+      @ok='deleteServerPool($event)'
+      @cancel='clearServerPoolToDelete'
+      @close='clearServerPoolToDelete'
+      ref='delete-server-pool-modal'
+      :static='modalStatic'>
+      <template v-slot:modal-title>
+        {{ $t('settings.serverPools.delete.title') }}
+      </template>
+      <template v-slot:modal-ok>
+        <b-spinner small v-if="deleting"></b-spinner>  {{ $t('app.yes') }}
+      </template>
+      <span v-if="serverPoolToDelete">
+        {{ $t('settings.serverPools.delete.confirm', { name:serverPoolToDelete.name }) }}
+      </span>
+
+      <div v-if="deleteFailedRoomTypes">
+        <b-alert :show="true" variant="danger">{{ $t('settings.serverPools.delete.failed') }}
+          <ul>
+            <li v-for="roomType in deleteFailedRoomTypes" :key="roomType.id">
+              {{ roomType.description }}
+            </li>
+          </ul>
+        </b-alert>
+      </div>
+
+    </b-modal>
+  </div>
+</template>
+
+<script>
+import Base from '../../../api/base';
+import Can from '../../../components/Permissions/Can';
+import ActionsColumn from '../../../mixins/ActionsColumn';
+import env from '../../../env';
+
+export default {
+  components: { Can },
+  mixins: [ActionsColumn],
+
+  props: {
+    modalStatic: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  computed: {
+    tableFields () {
+      const fields = [
+        { key: 'id', label: this.$t('settings.serverPools.id'), sortable: true },
+        { key: 'name', label: this.$t('settings.serverPools.name'), sortable: true },
+        { key: 'server_count', label: this.$t('settings.serverPools.serverCount'), sortable: true }
```

Comment:
The sorting does not work since this field is not whitelisted in the controller method!